transcom/mymove
# [MB-16327] Update transportation offices

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16327)

## Summary

When https://github.com/transcom/mymove/pull/11310 was merged, migrations on staging failed due to foreign key constraint violations when updating the `duty_locations` table ([link](https://app.circleci.com/pipelines/github/transcom/mymove/56673/workflows/9aa275e5-f081-4f7d-a270-b35bd9a75c66/jobs/972261)).

However, the `transportation_offices` migration seemed to work without issue. To help unblock this ticket, this PR only migrates the transportation offices as described in the spreadsheet linked in the above Jira ticket.

The Python script in this PR is essentially the same as was in Scott's [original PR](https://github.com/transcom/mymove/pull/11310), with some stylistic Pythonic tweaks.

## Verification Steps for the Author

These are to be checked by the author.

- [x] Verified that the SQL output in this PR is identical to https://github.com/transcom/mymove/pull/11310, except for new, randomly-generated uuids
- [x] Verified that the Contact box still shows up normally for service members

### How to test

1. Verify that the `transportation_offices` and `addresses` tables are in a valid state, and that there are 3 more TOs and addresses added after the migration is run
2. Verify that the new TOs show up as expected on the frontend (from the admin panel?)

### Database

#### Any new migrations/schema changes:


## Screenshots
